### PR TITLE
ADCM-1438 Config DRF refactoring

### PR DESCRIPTION
**Pull request by a-alferov**

### CODE REVIEW

File: python/api/config/serializers.py [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+class HistoryCurrentPreviousConfigSerializer(serializers.Serializer):
+    history = serializers.SerializerMethodField()
+    current = serializers.SerializerMethodField()
+    previous = serializers.SerializerMethodField()
+
+    def get_history(self, obj):
+        view_name = 'config-history'
+        return ConfigURL(read_only=True, view_name=view_name).get_url(
+            obj, view_name, self.context['request'], format=None)
+
```
Comment (grey-kristy) [inline python/api/config/serializers.py:53]: You don't need to do it so complicated and use SerializerMethodField. Just write:

history = ConfigURL(read_only=True, view_name=view_name)

File: python/api/config/views.py [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+    serializer_class = serializers.HistoryCurrentPreviousConfigSerializer
+    object_type = None
+
+    def get_queryset(self):
+        return get_objects_for_config(self.object_type)
+
+    def get(self, request, *args, **kwargs):
+        object_type = kwargs['object_type']
+        self.object_type = object_type
+        object_id = kwargs[f'{object_type}_id']
```
Comment (grey-kristy) [inline python/api/config/views.py:73]: This 3 string repeated too much. I suggest to write a function

object_type, object_id = get_obj_type_and_id(kwargs)

**Review by grey-kristy — CHANGES_REQUESTED** [review]

File: python/api/config/serializers.py (hunk 1 above)
Comment (a-alferov) [reply python/api/config/serializers.py]: Fixed in [https://github.com/arenadata/adcm/pull/563/commits/2ff1cbcf6ee5573650cd2fdf0928c4ac22360223](url)

File: python/api/config/views.py (hunk 2 above)
Comment (a-alferov) [reply python/api/config/views.py]: Fixed in https://github.com/arenadata/adcm/pull/563/commits/2ff1cbcf6ee5573650cd2fdf0928c4ac22360223

**Review by grey-kristy — APPROVED** [review]

**Review by nagarin — APPROVED** [review]

**Pull request merged**